gips/data/merra/merra.py

```diff
@@ -24,6 +24,15 @@
 import os
 import datetime
 import time
+
+from gips.data.merra.install_basic_client import install_basic_client
+
+URI = "urs.earthdata.nasa.gov"
+USER = "bobbyhbraswell"
+PASSWD = "Coffeedog_2"
```

Comment:
Hardcoded auth creds, looks like something got committed by mistake? (gah sorry for repeated comment deletions here, they repurposed the rightmost button recently)